src/engine/DnCWorker.h

```diff
@@ -36,7 +36,7 @@ class DnCWorker
       Pop one subQuery, solve it and handle the result
       Return true if the DnCWorker should continue running
     */
-    void popOneSubQueryAndSolve();
+    void popOneSubQueryAndSolve( bool restoreTreeStates=true );
```

Comment:
spaces: restoreTreeStates = true

Reply:
also,
align the `restoreTreeStates` with the default option